include/spdlog/details/registry.h

```diff
@@ -85,7 +85,10 @@ class registry
         formatter_ = std::move(formatter);
         for (auto &l : loggers_)
         {
-            l.second.lock()->set_formatter(formatter_->clone());
+            auto shared = l.second.lock();
+            if (shared) {
```

Comment:
Nit: Looks like the spdlog coding standard puts opening braces on their own line.